graph2tac/tf2/train.py

```diff
@@ -20,13 +20,15 @@
 from dataclasses_json import dataclass_json
 import yaml
 
+import graph2tac.loader.py_data_server   # import capnp before tensorflow
```

Comment:
Just so I know, why is it important to import capnp before tensorflow?  (Or is it just that tensorflow should be delayed since it takes so long to import?)

Reply:
On Cholesky cluster, it crashes otherwise (for not being able to find the linked libraries). I don't understand it properly, just know that it is not the first time when something had to be loaded before tensorflow.

Reply:
Have you tested the predict server?  Do we have to do the same there?

Reply:
Oh, that probably already loads capnp before TF actually.